containers/osbuild-composer/entrypoint.py

```diff
@@ -82,6 +88,19 @@ def _parse_args(self):
             dest="remote_worker_api",
             help="Disable the remote-worker-API",
         )
+        self._parser.add_argument(
+            "--remote-worker-api-port",
+            type=int,
+            default=8700,
+            dest="remote_worker_api_port",
+            help="Port which the remote-worker API listens on",
+        )
+        self._parser.add_argument(
```

Comment:
What about making this more generic? Basically accepting `address` as an argument, set to `::` by default.

Reply:
Done